### PR TITLE
Google BigQuery docs

### PR DESCRIPTION
**Pull request by ivarpruijn**
Adds documentation on how to set up and use Google BigQuery.

To do:
- [x] Add how to make a connection to a BQ database in 'getting started in your notebook'.
- [x] Add redirects for URLs that have changed/been removed.

### CODE REVIEW

File: docs/docs/tracking/collector/introduction.md [hunk 1]
```diff
@@ -1,44 +1,51 @@
...
+The Collector is self-hosted on your own domain, so no data is ever sent to any third-party, meaning:
+
+* You have full control over your data.
+* Tracking is compliant with privacy legislation such as GDPR, CCPA and PECR.
+* Ad blockers can be avoided: first-party data tracking is usually not covered by ad blockers.
+
+## Validation
+The Collector validates any incoming Event against the 
+[open analytics taxonomy](/taxonomy/reference/events/overview.md). If it fails, the Collector will respond 
+with an error, and store the Event in the `NOK` folder on disk.
```
Comment (jansentom) [inline docs/docs/tracking/collector/introduction.md:38]: This is a bit too on-disk-storage specific. How about:
```suggestion
with an error, and store the Event in the configured `NOK` (not-OK) location.

```

File: docs/docs/tracking/collector/google-bigquery.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+---
+sidebar_position: 3
+title: Google BigQuery
+slug: /tracking/collector/google-bigquery
+---
+
+# Google BigQuery
+
+The [Objectiv Collector](./introduction.md) can be configured to work with [Snowplow](./snowplow-pipeline.md) 
+on Google Cloud Platform and BigQuery. The Snowplow GCP setup uses GCP PubSub topics (message queue) to
```
Comment (jansentom) [inline docs/docs/tracking/collector/google-bigquery.md:10]:
```suggestion
The [Objectiv Collector](./introduction.md) can be configured to work with BigQuery through [Snowplow](./snowplow-pipeline.md) on Google Cloud Platform. The Snowplow GCP setup uses GCP PubSub topics (message queue) to 
```

Comment (jansentom) [reply docs/docs/tracking/collector/google-bigquery.md:10]: Similar applies to S3 story. It's about S3, not about SP or AWS imho

File: docs/docs/tracking/collector/snowplow-pipeline.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 5
+title: Snowplow pipeline
+slug: /tracking/collector/snowplow-pipeline
+---
+
+# Snowplow pipeline
+
+The Objectiv Collector supports using the Snowplow pipeline as a sink for Objectiv events, hooking directly 
+into Snowplows enrichment step. Currently, there is data store support for:
```
Comment (jansentom) [inline docs/docs/tracking/collector/snowplow-pipeline.md:10]: There is support for only BQ, but we have a tutorial to also write stuff to S3. In principle, this is no different then snowflake or databricks imho, those will work too with SP, but we can't query that yet.

File: docs/docs/modeling/bach/examples.mdx [hunk 4]
```diff
@@ -296,7 +303,7 @@ this case.
 :::caution important
 
 `df.sort_index().loc['Sleat':, 'municipality'] = 'Fryslân'` will have no effect on `df`, since
-`df.sort_index()` returns a new dataframe.
+[`sort_index()`](/modeling/bach/api-reference/DataFrame/bach.DataFrame.sort_index.mdx) returns a new dataframe.
```
Comment (KathiaBarahona) [inline docs/docs/modeling/bach/examples.mdx:306]:
```suggestion
[`sort_index()`](/modeling/bach/api-reference/DataFrame/bach.DataFrame.sort_index.mdx) returns a new Dataframe.
```

Comment (ivarpruijn) [reply docs/docs/modeling/bach/examples.mdx]: Thanks, fixed in https://github.com/objectiv/objectiv-analytics/pull/1029/commits/dec9351555f4c78dfb89322792713b2fab8f1edd and https://github.com/objectiv/objectiv.io/pull/492/commits/52ce514ca6b42fa72b31478437476320416dfb6c

File: docs/docs/modeling/bach/api-reference/Series/bach.Series.min.mdx [hunk 5]
```diff
@@ -33,6 +33,12 @@ a new Series with the aggregation applied
 
 
 
+#### Return type
+
+T
```
Comment (KathiaBarahona) [inline docs/docs/modeling/bach/api-reference/Series/bach.Series.min.mdx:38]: mmm here it should be `bach.series.series.T`

Comment (KathiaBarahona) [reply docs/docs/modeling/bach/api-reference/Series/bach.Series.min.mdx:38]: **question** Should we rename T to something more self-explanatory? Maybe `SeriesSubType`...? not sure

Comment (ivarpruijn) [reply docs/docs/modeling/bach/api-reference/Series/bach.Series.min.mdx]: Fixed in https://github.com/objectiv/objectiv.io/pull/492/commits/77d5668f90f78383a457612d13ca0c6c7b218cb6

**Pull request merged**